src/sidebar-container.component.ts

```diff
@@ -177,6 +178,24 @@ export class SidebarContainer implements AfterContentInit, OnChanges, OnDestroy
   }
 
   /**
+   * @internal
+   *
+   * Closes sidebars when the backdrop is clicked, if they have the
+   * `closeOnClickBackdrop` option set.
+   */
+  _onBackdropClicked(): void {
+    const _sidebars = this._sidebars.toArray();
```

Comment:
I see you just reused some of the code from below. I had to use an array for this so I could prematurely break from it if needed. In this case, we'll always need to hit every sidebar, so it could be written like the following:

```typescript
for (const sidebar in sidebars) {
  if (sidebar.opened && sidebar.showBackdrop && sidebar.closeOnClickBackdrop) {
    sidebar.close();
  }
}
```

I actually forget if it's `for-in` or `for-of` though, so you'll need to double check that.